src/node/entry.ts

```diff
@@ -67,7 +77,7 @@ const main = async (args: Args): Promise<void> => {
   const httpServer = new HttpServer(options)
   const vscode = httpServer.registerHttpProvider("/", VscodeHttpProvider, args)
   const api = httpServer.registerHttpProvider("/api", ApiHttpProvider, httpServer, vscode, args["user-data-dir"])
-  const update = httpServer.registerHttpProvider("/update", UpdateHttpProvider, !args["disable-updates"])
+  const update = httpServer.registerHttpProvider("/update", UpdateHttpProvider, true)
```

Comment:
Should we default this to false? I think that might be better since there's no way to disable it now.

Reply:
I thought we were disabling it?

Reply:
Oh, I think `true` means to enable it.

Reply:
Oh lol yup, my bad, didn't notice the flip.